src/neighbors.cc

```diff
@@ -984,7 +984,7 @@ neighborsUdpAck(const cache_key * key, icp_common_t * header, const Ip::Address
 
     debugs(15, 6, "neighborsUdpAck: opcode " << opcode << " '" << storeKeyText(key) << "'");
 
-    if (NULL != (entry = Store::Root().get(key)))
+    if (NULL != (entry = Store::Root().get(Store::CacheKey(key))))
```

Comment:
We should not need this horrible "NULL != " construct any more. if((entry = foo)) should be sufficient and correct.

Reply:
Fixed.